src/app/compare/compare.component.ts

```diff
@@ -34,8 +51,16 @@ export class CompareComponent implements OnInit, AfterViewInit {
     }, 100);
   }
 
+  getIdsFromPath() {
+    return <string>this.route.snapshot.paramMap.get('id');
```

Comment:
For consistency use the `var as Type` type assertion instead of `<Type>var`
```suggestion
    return this.route.snapshot.paramMap.get('id') as string;
```

Reply:
Fixed